bin/jsonschema_suite

```diff
@@ -88,12 +90,17 @@ class SanityTests(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         print(f"Looking for tests in {SUITE_ROOT_DIR}")
+        print(f"Looking for output tests in {OUTPUT_ROOT_DIR}")
         print(f"Looking for remotes in {REMOTES_DIR}")
 
         cls.test_files = list(collect(SUITE_ROOT_DIR))
         assert cls.test_files, "Didn't find the test files!"
         print(f"Found {len(cls.test_files)} test files")
 
+        cls.output_test_files = list(collect(OUTPUT_ROOT_DIR))
+        assert cls.remote_files, "Didn't find the output test files!"
```

Comment:
You have a typo here, which is the (first, though maybe not only) reason CI fails. You want

```suggestion
        assert cls.output_test_files, "Didn't find the output test files!"
```